node.gyp

```diff
@@ -70,8 +70,9 @@
       'lib/zlib.js',
       'lib/internal/child_process.js',
       'lib/internal/freelist.js',
-      'lib/internal/socket_list.js',
+      'lib/internal/linklist.js',
       'lib/internal/repl.js',
+      'lib/internal/socket_list.js',
```

Comment:
Almost everything was in alphabetical order (ignoring leading `_` chars). I don't feel strongly about it and am happy to move it back if need be.